### PR TITLE
feat: add .get_alias() method to the client

### PR DESCRIPTION
**Pull request by lilydjwg**
Let me try to fix my own issue https://github.com/matrix-org/matrix-rust-sdk/issues/673.

The method is named after ruma. I don't know if it should be kept the same but if not, I can give it a better name.

### CODE REVIEW

File: crates/matrix-sdk/src/client/mod.rs [hunk 1]
```diff
@@ -65,10 +66,10 @@ use ruma::{
     events::room::MediaSource,
     presence::PresenceState,
     MxcUri, OwnedDeviceId, OwnedRoomId, OwnedServerName, OwnedUserId, RoomId, RoomOrAliasId,
-    ServerName, UInt,
+    ServerName, UInt, RoomAliasId,
 };
 use serde::de::DeserializeOwned;
-#[cfg(not(target_arch = "wasm32"))]
+#[cfgnot(target_arch = "wasm32")]
```
Comment (jplatte) [inline crates/matrix-sdk/src/client/mod.rs:72]:
```suggestion
#[cfg(not(target_arch = "wasm32"))]
```

**Review by jplatte — APPROVED** [review]
I don't really care whether it's called `get_alias` or `resolve_alias`, but the code looks good to me 👍🏼

**Review by poljar — CHANGES_REQUESTED** [review]
Looks good, but we need to make CI happy. You can run `cargo xtask fixup` for that.

**Comment by lilydjwg** [comment]
I'm sorry I made some stupid mistakes...I fixed it just now. I tried to run the tests locally but it used too much memory so I have to Ctrl-C.

**Comment by jplatte** [comment]
There's still some formatting inconsistencies, can you please run `cargo +nightly fmt` locally?

**Comment by jplatte** [comment]
Also, we're aware that testing the SDK takes way too much memory. We unfortuantely don't know exactly what's the problem, but maybe we should re-prioritize looking into that..

**Comment by lilydjwg** [comment]
cargo fmt has been run.

I'm still running cargo xtask fixup. It needs to install and compile quite a lot things.

**Comment by jplatte** [comment]
It looks like the test you added is failing.

**Comment by lilydjwg** [comment]
The URL was wrong. It should have been fixed (I've run it on a server).

Also I decided to give it a better name.

File: crates/matrix-sdk/src/client/mod.rs [hunk 2]
```diff
@@ -608,6 +609,20 @@ impl Client {
         self.store().get_room(room_id).and_then(|room| room::Left::new(self.clone(), room))
     }
 
+    /// Resolve a room alias to a room id and a list of servers which knows
```
Comment (jplatte) [inline crates/matrix-sdk/src/client/mod.rs:612]:
```suggestion
    /// Resolve a room alias to a room id and a list of servers which know
```

**Pull request merged**